sentry_sdk/integrations/bottle.py

```diff
@@ -42,7 +45,6 @@ class BottleIntegration(Integration):
     def __init__(self, transaction_style="endpoint"):
         # type: (str) -> None
 
-        TRANSACTION_STYLE_VALUES = ("endpoint", "url")
```

Comment:
Hmm is it worth changing this (and in the other integrations)? Could break downstream code.

Reply:
How would the downstream code depend on this?

Reply:
`BottleIntegration.TRANSACTION_STYLE_VALUES`

Reply:
it's not an attribute but a local var in init (that's why the linter complained)

Reply:
Yes, I misread it, sorry.